drivers/serial/Kconfig.wch_usart

```
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 Google LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+config UART_WCH_USART
+	bool "CH32V00x USART driver"
+	default y
+	depends on DT_HAS_WCH_USART_ENABLED
+	select SERIAL_HAS_DRIVER
```

I think that this is the code suggestion that completes @nordicjm 's other comment. Please correct me if I'm wrong (or simply elaborate with another comment that provides more than zero information gain).

```suggestion
	select SERIAL_HAS_DRIVER
	select PINCTRL
```

---

It is as I said, if the driver uses it, I'm reviewing build system parts, not driver code, the author should know what drivers need to use it and select it at driver level, if those drivers need it then those are the correct places